content/docs/concepts/csidriver/features/powermax.md

```diff
@@ -680,3 +680,90 @@ These are the Container Storage Modules not supported with NVMeTCP protocol:
 - Container Storage Modules Observability
 - Container Storage Modules Application Mobility
 - Metro Replication
+
+## Multiple Availability Zones
+
+Starting with CSM 2.14.0, the PowerMax CSI driver supports multiple availability zones (AZ) for block protocols. NFS is not supported at this time.
```

Comment:
Let's follow CSM release versioning, CSM 1.14. We should not refer to driver image versions.